sig-architecture/production-readiness.md

```diff
@@ -28,6 +27,30 @@ aspects of the process.
     happens if it is subsequently upgraded again?
   - Are there tests for this?
 * Scalability
+  - Will enabling / using the feature result in any new API calls?
```

Comment:
Most of the issues I remember would not have been caught by this in a planning stage because they explosion of calls was generally a bug.  Maybe you could describe fanout concerns, where the controller or server fans out by more than order of magnitude?

Reply:
Even fan-out of 1 is something we would like to know - that's my point. In 95% (or more?) it will not block the feature, but we want to be aware of it.
BTW - regarding fan-out, the second bullet is exactly about this:
`
API calls that may be triggered by changes of some Kubernetes resources (e.g. update object X based on changes of object Y)
`